orttraining/orttraining/include/runner/orttraining_c_api.h

```diff
@@ -0,0 +1,308 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) SignalPop LLC. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include <string.h>
+#include <runner\training_runner.h>
+
+using namespace onnxruntime;
+using namespace onnxruntime::common;
+using namespace std;
```

Comment:
This is a big no no especially in the header. You making this decision for all the code that is ever to include this header including the clients code. All 3rd party classes including std must be fully qualified. I.g. std::vector

Reply:
Yes this is a C API intended to mirror the header https://github.com/MyCaffe/onnxruntime/blob/master/include/onnxruntime/core/session/onnxruntime_c_api.h

I Agree with you - I will move these to the cc file.